v2v/rhv-upload-plugin.py

```diff
@@ -67,11 +67,23 @@ def find_host(connection):
         debug("cannot read /etc/vdsm/vdsm.id, using any host: %s" % e)
         return None
 
-    debug("hw_id = %r" % vdsm_id)
+    system_service = connection.system_service()
+    storage_name = params['output_storage']
+    data_centers = system_service.data_centers_service().list(
+        search='storage=%s' % storage_name,
+        case_sensitive=False,
+    )
+    if len(data_centers) == 0:
+        # The storage domain is not attached to a datacenter
+        # (shouldn't happen, would fail on disk creation).
```

Comment:
I think we need a debug message here, similar to the one when vdsm_id is missing.